src/components/Newsletter/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styles from "../../styles/Newsletter.module.scss";
+
+const tab = -1;
+const newsletterValidation = (e) => {
```

Comment:
to do środka komponentu a sprawdzenia podobnie jak w przypadku slidera 
useRef() trzymasz referencje pól a nie takie cuda z pobieraniem elementu